doc/CHANGES_1_8.txt

```diff
@@ -41,3 +41,4 @@ WSDL parser / code generator changes, applying to both client and server side:
 * Added "hasValueFor{MemberName}()" accessor function, for optional elements
 * Generated services now include soapVersion() and endpoint() accessors to match the setSoapVersion(...) and setEndpoint(...) mutators
 * Added support for generating messages for WSDL files without services or bindings
+* Added proper support for QNames, which will allow the namespace to be extracted
```

Comment:
This needs to be much more detailed so people know what to expect. I'd mention that the API for serialize/deserialize has changed to take/return KDSoapValue instead of QVariant, for one thing. Anyone using that will have to port their code.

And then the generated code changed API when using xs:QName -- but I guess before your earlier commit this wouldn't compile at all?

This changelog paragraph should start with "Source incompatible change:" so people know that this might happen, like CHANGES_1_5.txt does.

Reply:
I replaced the changelog commit in this branch